### DISCUSSION TITLE
Granting Optional Permissions on Android

### DISCUSSION THREAD

**Discussion by Nilslag** [proginosko/LeechBlockNG#423] [Q&A]
Hi LeechBlock Users and Contributes!
I am using LeechBlock mainly on Android , and there seems to be no way to grant the optional Permission <all_urls> on this Platform!
And so the whole block Keywords Feature is practically not Usable on Android.

Does Anyone know a way to grant optional Permission? Or is a workaround, like some kind of ask for optional permissions button in LeechBlock, needed?

**Comment by proginosko** [comment]
Unfortunately, this is a known issue (#390 & #419) that cannot be resolved until support for Manifest V3 is improved on Android.
